x-pack/plugins/graph/public/application.ts

```diff
@@ -73,6 +73,7 @@ export interface GraphDependencies {
 
 export const renderApp = ({ appBasePath, element, kibanaLegacy, ...deps }: GraphDependencies) => {
   kibanaLegacy.loadFontAwesome();
+  kibanaLegacy.loadStyles();
```

Comment:
There seems to be a lot of individual loads of legacy styles. How does a plugin know if they need these or not? What constitutes `legacy` here?

Reply:
I think we don't need most of this anymore when using e.g. `src/legacy/ui/public/styles/_base.scss`

Reply:
migrated what's needed to  `src/core/public/styles` so there's no more need for `loadStyles`